lib/cerebrum.rb

```diff
@@ -1,5 +1,47 @@
 require "cerebrum/version"
+require 'matrix'
```

Comment:
also ps, when adding external dependencies you'll need to update the Gemfile so anyone can just `bundle install` and it'll pick up what they're missing. this is equivalent to `package.json` and `npm install`